[2024-05-04T11:35:22.809+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-04T11:35:22.925+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl.gcs_to_bq_geolocation scheduled__2024-05-03T00:00:00+00:00 [queued]>
[2024-05-04T11:35:23.074+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl.gcs_to_bq_geolocation scheduled__2024-05-03T00:00:00+00:00 [queued]>
[2024-05-04T11:35:23.079+0000] {taskinstance.py:2303} INFO - Starting attempt 3 of 3
[2024-05-04T11:35:23.222+0000] {taskinstance.py:2327} INFO - Executing <Task(GCSToBigQueryOperator): gcs_to_bq_geolocation> on 2024-05-03 00:00:00+00:00
[2024-05-04T11:35:23.322+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=9378) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-05-04T11:35:23.334+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'etl', 'gcs_to_bq_geolocation', 'scheduled__2024-05-03T00:00:00+00:00', '--job-id', '175', '--raw', '--subdir', 'DAGS_FOLDER/etl.py', '--cfg-path', '/tmp/tmpbyfs8jv1']
[2024-05-04T11:35:23.335+0000] {standard_task_runner.py:91} INFO - Job 175: Subtask gcs_to_bq_geolocation
[2024-05-04T11:35:23.331+0000] {standard_task_runner.py:63} INFO - Started process 9455 to run task
[2024-05-04T11:35:23.563+0000] {task_command.py:426} INFO - Running <TaskInstance: etl.gcs_to_bq_geolocation scheduled__2024-05-03T00:00:00+00:00 [running]> on host d4c1d9b975b8
[2024-05-04T11:35:24.933+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl' AIRFLOW_CTX_TASK_ID='gcs_to_bq_geolocation' AIRFLOW_CTX_EXECUTION_DATE='2024-05-03T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='3' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-05-03T00:00:00+00:00'
[2024-05-04T11:35:24.934+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-04T11:35:25.011+0000] {connection.py:274} WARNING - Connection schemes (type: google_cloud_platform) shall not contain '_' according to RFC3986.
[2024-05-04T11:35:25.043+0000] {base.py:84} INFO - Using connection ID 'my_gcp_conn' for task execution.
[2024-05-04T11:35:25.046+0000] {gcs_to_bigquery.py:379} INFO - Using existing BigQuery table for storing data...
[2024-05-04T11:35:25.495+0000] {bigquery.py:2328} INFO - Project is not included in destination_project_dataset_table: order.olist_geolocation_dataset; using project "project-capstone-422211"
[2024-05-04T11:35:25.495+0000] {gcs_to_bigquery.py:383} INFO - Executing: {'load': {'autodetect': True, 'createDisposition': 'CREATE_IF_NEEDED', 'destinationTable': {'projectId': 'project-capstone-422211', 'datasetId': 'order', 'tableId': 'olist_geolocation_dataset'}, 'sourceFormat': 'CSV', 'sourceUris': ['gs://storage-capstone/olist_geolocation_dataset.csv'], 'writeDisposition': 'WRITE_TRUNCATE', 'ignoreUnknownValues': False, 'skipLeadingRows': None, 'fieldDelimiter': ',', 'quote': None, 'allowQuotedNewlines': False, 'encoding': 'UTF-8'}}
[2024-05-04T11:35:25.496+0000] {bigquery.py:1613} INFO - Inserting job ***_etl_gcs_to_bq_geolocation_2024_05_03T00_00_00_00_00_92fd3d1abfcff43469e9333e7c0a8417
[2024-05-04T11:36:04.442+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-04T11:36:04.519+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=etl, task_id=gcs_to_bq_geolocation, execution_date=20240503T000000, start_date=20240504T113522, end_date=20240504T113604
[2024-05-04T11:36:04.704+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-04T11:36:04.838+0000] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-05-04T11:36:04.850+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
